packages/atomic/src/locales.json

```diff
@@ -809,6 +809,114 @@
     "zh-CN": "晴朗",
     "zh-TW": "清除"
   },
+  "filter-removed": {
+    "en": "Filter removed",
+    "fr": "Filtre supprimé",
+    "cs": "Filtr odstraněn",
+    "da": "Filter fjernet",
+    "de": "Filter entfernt",
+    "el": "Το φίλτρο αφαιρέθηκε",
+    "es": "Filtro eliminado",
+    "fi": "Suodatin poistettu",
+    "hu": "Szűrő eltávolítva",
+    "id": "Filter dihapus",
+    "it": "Filtro rimosso",
+    "ja": "フィルターが削除されました",
+    "ko": "필터가 제거되었습니다",
+    "nl": "Filter verwijderd",
+    "no": "Filter fjernet",
+    "pl": "Filtr usunięty",
+    "pt": "Filtro removido",
+    "pt-BR": "Filtro removido",
+    "ru": "Фильтр удалён",
+    "sv": "Filter borttaget",
+    "th": "ลบตัวกรองแล้ว",
+    "tr": "Filtre kaldırıldı",
+    "zh": "筛选器已移除",
+    "zh-CN": "筛选器已移除",
+    "zh-TW": "已移除篩選器"
+  },
+  "clear-search": {
```

Comment:
I think this string is a bit of an outlier in this PR:
We are more specific about what's being cleared by the action of the user, and here, `clear-search` is not IMO.
When I read "Clear search", I understand "Clear _all_ the search stuff", filters, sort options, and the like.
What's actually being clear when I click is the Search Box, so:
`clear-searchbox` would be the right key, and "Clear searchbox"/"Effacer la [barre de recherche](https://vitrinelinguistique.oqlf.gouv.qc.ca/fiche-gdt/fiche/17010896/barre-de-recherche)"[^1]

[^1]: Boîte de recherche, which is sometimes visible in Canadian French, is a [loan translation](https://vitrinelinguistique.oqlf.gouv.qc.ca/23822/les-emprunts-a-langlais/emprunts-idiomatiques/quest-ce-quun-emprunt-idiomatique).